client/electron/package.js

```diff
@@ -92,6 +92,13 @@ const options = {
 
   win: {
     target: 'nsis',
+    extraResources: [
+      {
+        from: 'node_modules/regedit/vbs',
+        to: 'vbs',
+        filter: ['**/*'],
+      },
+    ],
```

Comment:
Note that modules listed under `dependencies` in `package.json` are included in the application

(that also mean we include regedit, on Linux / MacOS)

Reply:
I would also note that you are including `vbs` 2 time

Reply:
I don't know if there's an easy way to add a dependency for one OS only. Since I import the package only on Windows it's probably not much of a problem.